graylog2-server/src/main/java/org/graylog/scheduler/JobSchedulerConfiguration.java

```diff
@@ -27,11 +28,19 @@
 @SuppressWarnings({"FieldCanBeLocal", "unused", "WeakerAccess"})
 public class JobSchedulerConfiguration implements PluginConfigBean {
     public static final String LOOP_SLEEP_DURATION = "job_scheduler_loop_sleep_duration";
+    public static final String LOCK_EXPIRATION_TIME_MINUTES = "job_lock_expiration_time_minutes";
```

Comment:
How about using the same prefix as the other option for consistency?
```suggestion
    public static final String LOCK_EXPIRATION_TIME_MINUTES = "job_scheduler_lock_expiration_time_minutes";
```